operator/controllers/kyma_controller.go

```diff
@@ -53,6 +56,8 @@ type RequeueIntervals struct {
 	Waiting time.Duration
 }
 
+const componentName = "kyma"
```

Comment:
can we make it so that it instead uses Kind? then theres already a constant available in types